### DISCUSSION TITLE
Lock file already exists problem

### DISCUSSION THREAD

**Discussion by TG-009** [community/community#42405] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am using git hub desktop and I am getting this message no matter what I do "A Lock file already exists in the repository, which blocks this operation from completing."
What do I do, I get this message every time I try to change branches, pull the origin.
HELP

**Comment by SilasRodrigues19** [comment]
Hi @TG-009 

There is a closed issue on GitHub Desktop repo about this topic in the link below:
[https://github.com/desktop/desktop/issues/5475](https://github.com/desktop/desktop/issues/5475)

I hope it helps you

**Reply by TG-009** [reply]
Thank you.
See also answer here  https://github.com/community/community/discussions/42409
